packages/web/index.js

```diff
@@ -30,7 +30,13 @@ module.exports = (neutrino, opts = {}) => {
     polyfills: {
       async: true
     },
-    babel: {}
+    babel: {},
+    minify: {
+      minify: {
+        removeConsole: true,
```

Comment:
Removing these would be a breaking change, just FYI.

Reply:
Sure. As a first step, we could just allow the `minify` option to be passed, and not set the defaults.

Otherwise changing it looks something like:

```
neutrino.use('neutrino-preset-web);
neutrino.config
.when(isProduction, (config) => {
  config.plugin('minify').tap(() => [{
    removeConsole: true,
    removeDebugger: true,
  }]);
});
```